guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -42,22 +42,68 @@ ifdef::katello[]
 ----
 # {package-manager} config-manager --set-enabled powertools
 ----
+
++
+. Enable the Katello module:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} module enable katello:el8
```

Comment:
```suggestion
# {package-manager} module enable katello:el8 pulpcore:el8
```

unless we decide to make katello depend on pulpcore, which we so far avoided.

Reply:
I was sure I checked if you could enable multiple modules at the same time on the CLI, but it looks like I didn't or I missed something.

Reply:
You can.